stake-pool/cli/src/main.rs

```diff
@@ -1,7 +1,9 @@
 mod client;
+mod output;
 
 use {
     crate::client::*,
+    crate::output::{CliStakePool, CliStakePools, CliStakePoolDetails, CliStakePoolStakeAccountInfo},
```

Comment:
```suggestion
    crate::{
        client::*, output::{CliStakePool, CliStakePools, CliStakePoolDetails, CliStakePoolStakeAccountInfo},
    },
```
(might need `cargo fmt`)